### ISSUE TITLE
labels dont work

### ISSUE THREAD

**Issue by sadath-12** [tensorlakeai/indexify#809]
Lets say I upload a content 

```sh
curl -X 'POST' \
  'http://ip:8900/namespaces/default/extraction_graphs/ppt2/extract' \
  -H 'accept: */*' \
  -H 'Content-Type: multipart/form-data' \
  -F 'file=@figures.pptx;type=application/vnd.openxmlformats-officedocument.presentationml.presentation' \
  -F 'labels={"additionalProp1":"string","additionalProp2":"string","additionalProp3":"string"}'
```

and when I try to search over 

```sh
curl -X 'POST' \
  'http://ip:8900/namespaces/default/indexes/ppt2.chunk_embeddings.embedding/search' \
  -H 'accept: application/json' \
  -H 'Content-Type: application/json' \
  -d '{
 
  "include_content": true,
  "k": 2,
  "query": "list content"
}'
```

I get response 

```json
{
  "results": [
    {
      "content_id": "fec28c0b85feed93",
      "text": "Dataset",
      "mime_type": "text/plain",
      "confidence_score": 0.21810222,
      "labels": {},
      "root_content_metadata": {
        "id": "4e40ffa897bbfcfc",
        "parent_id": "",
        "root_content_id": "",
        "namespace": "default",
        "name": "p6H8O6X2RQQMNJil.pptx",
        "mime_type": "application/vnd.openxmlformats-officedocument.presentationml.presentation",
        "labels": {},
        "extraction_graph_names": [
          "ppt2"
        ],
        "storage_url": "s3://indexify/p6H8O6X2RQQMNJil.pptx",
        "created_at": 1722581782,
        "source": "",
        "size": 13250823,
        "hash": "7604bfa4bf2d4b42c998485230bc4c846294155b7cf01052bd3403a387c333c3"
      },
      "content_metadata": {
        "id": "fec28c0b85feed93",
        "parent_id": "87ef79be1da15a7d",
        "root_content_id": "4e40ffa897bbfcfc",
        "namespace": "default",
        "name": "DKywNWwHzVuKBdAsZA8FP",
        "mime_type": "text/plain",
        "labels": {},
        "extraction_graph_names": [
          "ppt2"
        ],
        "storage_url": "s3://indexify/DKywNWwHzVuKBdAsZA8FP",
        "created_at": 1722581904,
        "source": "text_chunker",
        "size": 7,
        "hash": "4411e525721e7dd801755882fd2361b2c4fb90a8bd23b456782616601397ef4e"
      }
    },
    {
      "content_id": "76a90b64faec6b6b",
      "text": "Dataset",
      "mime_type": "text/plain",
      "confidence_score": 0.21810222,
      "labels": {},
      "root_content_metadata": {
        "id": "253f09cfa2c8aad6",
        "parent_id": "",
        "root_content_id": "",
        "namespace": "default",
        "name": "Xq8CcqdwVc2YA8-7.pptx",
        "mime_type": "application/vnd.openxmlformats-officedocument.presentationml.presentation",
        "labels": {},
        "extraction_graph_names": [
          "ppt2"
        ],
        "storage_url": "s3://indexify/Xq8CcqdwVc2YA8-7.pptx",
        "created_at": 1722581444,
        "source": "",
        "size": 13250823,
        "hash": "7604bfa4bf2d4b42c998485230bc4c846294155b7cf01052bd3403a387c333c3"
      },
      "content_metadata": {
        "id": "76a90b64faec6b6b",
        "parent_id": "254cd90909881b1",
        "root_content_id": "253f09cfa2c8aad6",
        "namespace": "default",
        "name": "l72bt8qIGNfyunxim_Dil",
        "mime_type": "text/plain",
        "labels": {},
        "extraction_graph_names": [
          "ppt2"
        ],
        "storage_url": "s3://indexify/l72bt8qIGNfyunxim_Dil",
        "created_at": 1722581544,
        "source": "text_chunker",
        "size": 7,
        "hash": "4411e525721e7dd801755882fd2361b2c4fb90a8bd23b456782616601397ef4e"
      }
    }
  ]
}
```

I thought I'm supposed to receive the labels when I search

**Comment by maxkozlovsky** [comment]
Looks like it works only if labels are specified before the file. I'll fix it so the order does not matter.

**Comment by sadath-12** [comment]
hmm.. @maxkozlovsky for example ?? the labels were always empty for me in retrieval , even if I specify it before upload of file . I'm using swagger UI for uploading

**Comment by maxkozlovsky** [comment]
Fixed in #819
